test/e2e-tests.sh

```diff
@@ -23,15 +23,19 @@ failed=0
 # Run serverless-operator specific tests.
 (( !failed )) && serverless_operator_e2e_tests || failed=2
 if [[ $TEST_KNATIVE_KAFKA == true ]]; then
-  (( !failed )) && install_strimzi || failed=6
-  (( !failed )) && serverless_operator_kafka_e2e_tests || failed=7
+  (( !failed )) && install_strimzi || failed=3
+  (( !failed )) && serverless_operator_kafka_e2e_tests || failed=4
 fi
 
-(( !failed )) && ensure_serverless_installed || failed=3
+(( !failed )) && ensure_serverless_installed || failed=5
 # Run Knative Serving & Eventing downstream E2E tests.
 (( !failed )) && downstream_serving_e2e_tests || failed=4
 (( !failed )) && downstream_eventing_e2e_tests || failed=5
```

Comment:
Oh, I realized these numbers are dupe. Not an important thing at all. Only consider changing if you need to do something. Otherwise, it is not worth the 2 hour CI wait

Reply:
I think I fixed this, thanks for the heads up (had to make other changes, lumped in these as well)